index.html

```diff
@@ -3,10 +3,10 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Microverse</title>
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
     <h1 class="headingStyle">Hello Microverse!</h1>
 </body>
```

Comment:
- [Optional] COnisder using an indentation of two spaces in all your documents. This is a convention and also best practice.

Reply:
Okay I will consider, Thank you